### DISCUSSION TITLE
Fréquence Îles

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#2185] [Idées acceptées / déjà implémentées]
Bonjour à tous !

Amitié ou performance, c'est souvent difficile de faire un choix :D j'ai une guilde de 2 joueurs quotidiens mais pas plus, 1 inactif.. je me retrouve seul à chercher l'île, cela fait 19 jours que je n'ai pas pu m'y rendre, pourtant je me suis focalisé dessus..

Je propose que la proba lié à l'île soit dépendante du nombre de joueurs dans la guilde. Pour une guilde de 6 joueurs, elle reste la même, mais augmente si moins de joueurs. Ça ne change donc rien pour les grosses guildes :)

Cela permettrait trois choses : 
1- rendre la situation plus équitable entre les petites guildes et les autres.
2- de faire en sorte que les guildes se débarrassent des joueurs inactifs servant à recup le bonus quotidien..
3- chercher l'île est bloquant pour l'arène, donc les grandes guildes ont plus vite fini les 3 tours et peuvent plus facilement se concentrer sur l'arène..

Le bonus lié à l'île est un véritable gamechanger, mais je ne me vois pas quitter mes amis pour une maj qui me confronte à ce choix :) La maj oblige de se regrouper avec des membres de guilde qui ont les mêmes aspirations..

Voilà voilà :) 

PS on pourrait déjà faire une regen auto au début de l'île ça permettrait de ne pas bloquer les gens vis à vis de l'arène :)

**Comment by Procion06** [comment]
C'est un vrai défi de faire l'arène et l'ile en même temps, si tu combats rip l'île, et si tu fais l'île, qu'il faut déjà trouver, rip les combats
la régen auto est une bonne idée mais tu ne peux pas la faire au début, par contre à la fin oui, plutôt qu'une régen vraiment partielle qui ne sert pas à grand chose

**Comment by BastLast** [comment]
ceci a été fix, puisque c'était un bug
